run/tesla_api.py

```diff
@@ -258,92 +257,14 @@ def list_vehicles():
     return _execute_request(base_url, None, None, False)
 
 
-def get_service_data():
```

Comment:
why did you delete all these?

Reply:
Seems like unused code and not relevant for the job of keeping the car awake so I removed it.
Can put it back if it's a blocker.

Reply:
I would prefer to keep these.